Appraisals

```diff
@@ -1556,6 +1587,11 @@ elsif ruby_version?('3.2')
     gem 'nokogiri', platform: :ruby # TODO: binary gem has max ruby version constraint excluding previews, switch to using minimum version constraint once a non-3.2-excluding binary gem is released
   end
 
+  appraise 'sinatra' do
+    gem 'sinatra'
+    gem 'rack-test'
+  end
```

Comment:
Minor: Should we add a `>= 3` to this block on the supported Rubies, so that we ensure that we get the intended Sinatra version? (It's not a problem now, but I'm thinking this could be a just-in-case other dependency starts preventing the intended version from being picked up)

Reply:
Adding a minimum version doesn't harm, and makes gem resolution faster, so I agree here.